scripts/setup-project.sh

```
@@ -0,0 +1,750 @@
+    local key=$1
+    local value=$2
+    local scope=${3:-"production"}
+
+    # Update .env file
+    sed -i '' "s|^${key}=.*|${key}=${value}|" .env
+    
+    # Set in Vercel
+    echo -e "${BLUE}Setting $key in Vercel...${NC}"
+    
+    # Create a temporary file for the value
+    local tmp_file=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)